tools/ci_build/github/azure-pipelines/templates/android-qnn-java-api-aar.yml

```diff
@@ -0,0 +1,115 @@
+parameters:
+- name: buildConfig
+  displayName: Configuration to build
+  type: string
+  default: 'Release'
+
+- name: buildSettings
+  displayName: Path to the build setting file
+  type: string
+  default: ''
+
+- name: includedOpsConfig
+  displayName: Path to the included ops config file
+  type: string
+  default: ''
```

Comment:
I don't think we need this anymore. It was an historical thing when we built 'mobile' packages with limited operators and types.